### ISSUE TITLE
use_test_for_validation - no longer needed?

### ISSUE THREAD

**Issue by tpietruszka** [n-waves/ulmfit-multilingual#24]
In `ulmfit/train_clas.py` there is a somewhat confusing setting `use_test_for_validation`. 

Am I correct to assume that it's just a work-around for this bug? 
https://github.com/fastai/fastai/issues/1292

If so, the proper fix has been merged a few days ago: https://github.com/fastai/fastai/pull/1293

In such a case it could be useful to get rid of this work-around. I can give it a try, but only after a confirmation that it is not needed for other reasons.

**Comment by PiotrCzapla** [comment]
It is implemented like that to be able to compare with the fastai notebooks as they are using Test set for validation. Besides as far as I remember the constructors for text databunches were ignoring labels for testset, I'm not sure if it is fixed already in fastai. Create a PR if the code can be simplified.

**Comment by PiotrCzapla** [comment]
I've removed that in https://github.com/n-waves/ulmfit-multilingual/tree/multifit
